main/opcodes/create-terminate-context.zkasm

```diff
@@ -531,9 +530,11 @@ opDELEGATECALL:
 
     $ => B                   :MLOAD(lastMemOffset)
     $ => A                   :MLOAD(addrCall)
-    ${isWarmedAddress(A)} => C
+    D => E
```

Comment:
Maybe we can set retCallLength from D before checking if is warmed so we can avoid this swap ( D => E, E => D)